src/materials/ConcreteThermalMoisture.C

```diff
@@ -59,15 +58,14 @@ ConcreteThermalMoisture::validParams()
   params.addParam<MooseEnum>(
       "aggregate_pore_type", aggregate_pore_type, "aggregate pore structure");
 
-  MooseEnum moisture_diffusivity_model("Bazant Xi Mensi", "Bazant");
+  MooseEnum moisture_diffusivity_model("Bazant Mensi", "Bazant");
   params.addParam<MooseEnum>(
       "moisture_diffusivity_model", moisture_diffusivity_model, "moisture diffusivity models");
 
   params.addParam<Real>("cement_mass", 354.0, "cement mass (kg) per m^3");
   params.addParam<Real>("aggregate_mass", 1877.0, "aggregate mass (kg) per m^3");
   params.addParam<Real>("water_to_cement_ratio", 0.43, "water to cement ratio");
   params.addParam<Real>("concrete_cure_time", 23.0, "concrete curing time in days");
-  params.addParam<Real>("aggregate_vol_fraction", 0.7, "volumetric fraction of aggregates");
 
   // parameters for Bazant mositure diffusivity model
   params.addParam<Real>("D1", 3.0e-10, "empirical constants");
```

Comment:
We can add unit in the parameter description. It should be m^2/sec

Reply:
I agree with both of these comments, but I think they're both better handled in separate PRs. I was trying to focus on testing the material models and deleting the ones that have issues in this one. We still don't have testing of the PDE solution in this PR, but I was planning on a follow-on PR that removes the moisture capacity from `ConcreteMoistureTimeIntegration` and adds a revised version of the Maqbeth test. I was assuming the documentation cleanup (including parameter descriptions) would be handled in your PR.

Reply:
Okay. I will submit a PR for these changes once this is merged.